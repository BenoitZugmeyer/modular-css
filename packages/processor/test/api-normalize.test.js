"use strict";

const namer = require("@modular-css/test-utils/namer.js");
const relative = require("@modular-css/test-utils/relative.js");

const Processor = require("../processor.js");

describe("/processor.js", () => {
    describe(".normalize()", () => {
        let processor;

        beforeEach(() => {
            processor = new Processor({
                namer,
            });
        });

        it("should normalize inputs", async () => {
            expect(
                relative([ processor.normalize("../modular-css/simple.css") ])
            ).toMatchSnapshot();
        });
    });
});
