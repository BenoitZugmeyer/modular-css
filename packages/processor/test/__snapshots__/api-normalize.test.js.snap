// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/processor.js .normalize() should normalize inputs 1`] = `
Array [
  "simple.css",
]
`;
